rollbar-cli/package.yaml

```
@@ -5,7 +5,7 @@ license: MIT
 author: "Stack Builders Inc."
 maintainer: "David Mazarro <dmunuera@stackbuilders.com>"
 copyright: "2020-present Stack Builders Inc."
-tested-with: GHC ==8.8.4, GHC ==8.10.7, GHC ==9.4.7
+tested-with: GHC ==9.4.7, GHC ==9.6.7, GHC ==9.8.4, GHC ==9.10.1, GHC ==9.12.2
```

@davidnbr I believe this would significantly slow down the build, as we are adding 2 new versions. @DavidMazarro said that maintaining support for 9.6 is important, so we could use that as a starting point and then add the next 2 minor versions. Something like:


```suggestion
tested-with: GHC ==9.6.7, GHC ==9.8.4, GHC ==9.10.1
```

What are your thoughts?